performance/config.py

```diff
@@ -16,6 +16,8 @@
 class Config:
     ENV = 'prod'
 
+    REPORTS_S3_BUCKET = 'govukverify-hub-prod-billing-reports'
+    S3_VERIFICATIONS_DIRECTORY = 'rp'
     VERIFY_DATA_PIPELINE_CONFIG_PATH = os.path.abspath(
```

Comment:
I've been thinking about how we might deploy this, and I'm wondering whether we might need to make this directory overridable in the environment. Might want to talk this through in person, and this might not be the PR to do it in anyway.